src/vs/workbench/contrib/codeEditor/browser/find/simpleFindWidget.css

```diff
@@ -45,6 +46,7 @@
 
 .monaco-workbench .simple-find-part .monaco-findInput {
 	flex: 1;
+	min-width: 220px;
```

Comment:
Should this use flex-grow instead so it bases the width off its container?

Reply:
I tried that and could be doing something, but it didn't work